.github/workflows/docker/compose/nginx-compose.yaml

```diff
@@ -5,5 +5,5 @@
 services:
   nginx:
     build:
-      dockerfile: comps/nginx/Dockerfile
+      dockerfile: comps/3rd_parties/nginx/src/Dockerfile
```

Comment:
duplicate image, should be in `.github/workflows/docker/compose/3rd_parties-compose.yaml`